src/Psl/Xml/Issue/Level.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml\Issue;
+
+use Psl;
+use Psl\Exception\InvariantViolationException;
+
+use function in_array;
+
+use const LIBXML_ERR_ERROR;
+use const LIBXML_ERR_FATAL;
+use const LIBXML_ERR_NONE;
+use const LIBXML_ERR_WARNING;
+
+/**
+ * @psalm-immutable
+ */
+final class Level
+{
+    private const LEVEL_ERROR = LIBXML_ERR_ERROR;
+    private const LEVEL_FATAL = LIBXML_ERR_FATAL;
+    private const LEVEL_NONE = LIBXML_ERR_NONE;
```

Comment:
should be removed after the suggested changes.

Reply:
We still need the values. Hardcoded them as numbers but checked if they matched the once from libxml inside the testsuite. That makes it libxml-independant.